src/element_handle.ts

```diff
@@ -1,6 +1,8 @@
+/// <reference lib="dom" />
```

Comment:
This is a little scary. Do we need this?

Reply:
I checked, we can delete it

Reply:
Does this implicitly add the dom reference to downstream consumers? If so, we can't really do this.

Reply:
I have no idea how to solve this

Reply:
We don't need to strongly type the argument of the evaluate function. Making it `(element: unknown, ...` would be preferable to forcing all downstream consumers to use the dom reference.